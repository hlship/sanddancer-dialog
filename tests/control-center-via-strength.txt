%% Test roof to control center using strength
(setup lighting)
teleport roof
n
u
climb barrel
push barrel
(now) (knock holds #strength)
push barrel
l
push barrel
climb barrel
d
u

