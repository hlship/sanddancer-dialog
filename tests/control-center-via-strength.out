

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> (setup lighting)
Query succeeded: (setup lighting)
> teleport roof

Roof
From the roof of the building, you can see the huge electrical tower rising up
in front of you. Some metal rungs once led to a control room higher up the
tower, but the lowest ones are well above your head. You'd have to jump out into
space to grab the bottom rung. A huge metal barrel stands upright and intact
nearby, large enough that you could climb up on it. At your feet is the way back
down.

> n
It's a drop off on all sides.

> u
The metal rungs have rusted away and crumbled; the lowest are well above your
head. You don't have the courage to make the jump.

> climb barrel
Yeah, you could get on top of it, but while it's over here, you couldn't reach a
whole lot.

> push barrel
You push against the side, but you're not strong enough to move the barrel.

You shiver in the cold.

> (now) (knock holds #strength)
> push barrel
Heaving, you push against the barrel, and with a horrible grating noise, it
slides across the roof till it rests under the metal rungs.

> l
Roof
From the roof of the building, you can see the huge electrical tower rising up
in front of you. Some metal rungs once led to a control room higher up the
tower, but the lowest ones are well above your head. You'd have to jump out into
space to grab the bottom rung. A huge metal barrel stands upright and intact
nearby, directly underneath the the bottom rungs. At your feet is the way back
down.

> push barrel
Push it any more and it'll fall off the roof. Keep it here and you can get to
the rungs.

> climb barrel
You climb the barrel, reach, grab the bottom rungs, and pull yourself up.

Control Center
Some kind of electrical control stuff must have happened here once, but it's
totally trashed now. Like, piles of trash, broken equipment, and junk are
everywhere.

But hey, on one pile there's a perfectly good can opener.

Back down is the only place to go.

> d
You climb down.

Roof
From the roof of the building, you can see the huge electrical tower rising up
in front of you. Some metal rungs once led to a control room higher up the
tower, but the lowest ones are well above your head. You'd have to jump out into
space to grab the bottom rung. A huge metal barrel stands upright and intact
nearby, directly underneath the the bottom rungs. At your feet is the way back
down.

> u
You climb the barrel, reach, grab the bottom rungs, and pull yourself up.

Control Center
Some kind of electrical control stuff must have happened here once, but it's
totally trashed now. Like, piles of trash, broken equipment, and junk are
everywhere.

But hey, on one pile there's a perfectly good can opener.

Back down is the only place to go.

> 
